docs/Navigation/SubscribeWayPoints/index.md

```diff
@@ -15,6 +15,10 @@ Purpose
 Send future waypoint updates to SDL using [Navigation.OnWayPointChange](../onwaypointchange)
 !!!
 
+!!! note
```

Comment:
I don't understand how 3 times is counted in `during 3 consecutive ignition cycles.`
please correct me if I'm wrong

#original ignition 
app send RPC `SubscribeWayPoints` and get a success, 
SDL save the way points subscription status as part of resumption data.
#1st ignition
ignition off and on again.
app send RPC registerAppInterface(hashid)
SDL restore status of subscription. count =1
resumption data not changed during this ignition. hashid remain the same.
#2nd ignition
ignition off and on again.
app send RPC registerAppInterface(same hashid)
SDL restore status of subscription. count =2.  (if hashid every changes during last ignition count=1)
resumption data not changed during this ignition. hashid remain the same.
#3rd ignition
ignition off and on again.
app send RPC registerAppInterface(same hashid)
SDL restore status of subscription. count =3.  (if hashid every changes during last ignition count=1)
#4th ignition
ignition off and on again.
app send RPC registerAppInterface(same hashid)
SDL does not restore status of subscription, because count=4

Reply:
what will happen if during 1st / 2nd or 3rd ignition(s), app does not send registerAppInterface(hashid)? 
somehow count shall still be  
count=1 for 1st ignition, 
count=2 for 2nd ignition 
and count=3 for 3rd ignition
if #4th ignition
ignition off and on again.
count =4
app send RPC registerAppInterface(same hashid)
SDL does not restore status of subscription, because count=4

Reply:
if app does not register during 3 ign consecutive cycles ->then SDL deletes app's related data from resumption list;
if app registers -> then the counter resets